### PR TITLE
Fix CODEOWNERS string

### PR DESCRIPTION
**Pull request by arsulegai**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # This file is to automatically add code owners in PR reviews
-* @hyperledger-updates-maintaineres
+* @hyperledger-tooling/hyperledger-updates-maintaineres
```
Comment (ryjones) [inline CODEOWNERS]: _maintainers_ not _maintaineres_

**Review by ryjones — CHANGES_REQUESTED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (arsulegai) [reply CODEOWNERS]: I copied this text from the GitHub

image

Comment (ryjones) [reply CODEOWNERS]: I renamed it to fix the typo, while this was in flight.

Comment (arsulegai) [reply CODEOWNERS]: Fixed, thanks

**Review by ryjones — APPROVED** [review]

**Pull request merged**